BangazonWorkforce/Views/Employee/Index.cshtml

```diff
@@ -1,15 +1,35 @@
 ﻿@model IEnumerable<BangazonWorkforce.Models.Employee>
+<<<<<<< HEAD
```

Comment:
You have some merge conflicts here-- please clean these up before anyone reviews 👍